### PR TITLE
0.34.1 blog post

### PR DESCRIPTION
**Pull request by devlead**

### CODE REVIEW

File: input/blog/2019-07-17-cake-v0.34.1-released.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿---
+title: Cake v0.34.1 released
+category: Release Notes
+author: devlead
+---
+
+Version 0.34.1 of Cake has been released.
+
+This is just a minor release as a hotfix to address an issue breaking scripts when the dynamic C# keyword was used in build scripts. This was an side effect of updating from Roslyn 2.x to 3.x.
```
Comment (patriksvensson) [inline input/blog/2019-07-17-cake-v0.34.1-released.md]: Shouldn't this be `a side effect`?

File: input/blog/2019-07-17-cake-v0.34.1-released.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Thanks [dieterv](https://github.com/dieterv) for reporting, all contributions are appreciated❤.
```
Comment (patriksvensson) [inline input/blog/2019-07-17-cake-v0.34.1-released.md]: Perhaps `reporting. All contributions...`?

File: input/blog/2019-07-17-cake-v0.34.1-released.md (hunk 1 above)
Comment (devlead) [reply input/blog/2019-07-17-cake-v0.34.1-released.md]: yes I'll fix,

File: input/blog/2019-07-17-cake-v0.34.1-released.md (hunk 2 above)
Comment (devlead) [reply input/blog/2019-07-17-cake-v0.34.1-released.md]: sure

File: input/blog/2019-07-17-cake-v0.34.1-released.md (hunk 1 above)
Comment (devlead) [reply input/blog/2019-07-17-cake-v0.34.1-released.md]: fixed

File: input/blog/2019-07-17-cake-v0.34.1-released.md (hunk 2 above)
Comment (devlead) [reply input/blog/2019-07-17-cake-v0.34.1-released.md]: fixed

**Pull request merged**